src/System.Reflection.Primitives/ref/System.Reflection.Primitives.Forwards.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// These types were moved down to System.Runtime
```

Comment:
You don't need to create a new file for these, as they can be in System.Reflection.Primitives.cs as well.